xlrd/book.py

```diff
@@ -1159,7 +1159,7 @@ def handle_sst(self, data):
         # DEBUG = 1
         if DEBUG:
             print("SST Processing", file=self.logfile)
-            t0 = time.time()
+            t0 = perf_counter()
```

Comment:
Is this line change unrelated?

Reply:
If you're going to accept #305, I already make this change in my various cleanup and would appreciate not having to resolve git conflicts.

Reply:
Oh wait, you've already made this change!